core/dbt/include/global_project/docs/overview.md

```diff
@@ -38,6 +38,6 @@ Note that you can also right-click on models to interactively filter and explore
 - [What is dbt](https://docs.getdbt.com/docs/overview)?
 - Read the [dbt viewpoint](https://docs.getdbt.com/docs/viewpoint)
 - [Installation](https://docs.getdbt.com/docs/installation)
-- Join the [chat](https://community.getdbt.com/) on Slack for live questions and support.
+- Join the [dbt Community](https://www.getdbt.com/community/) for questions and discussion
```

Comment:
This is what's included by default in the dbt-docs site `Overview`, until/unless users provide a [custom overview](https://docs.getdbt.com/docs/building-a-dbt-project/documentation#setting-a-custom-overview). It could be simpler / have less info in it — is that what you're thinking?

Reply:
sure. but does the `global_project` need it? `global_project` doesn't show up in anyones dbt docs, nor are any of the macros documented? I guess this file is just present in all scaffolded dbt projects?

Reply:
That's right - this is how it gets into users' project, by default.

It's an opinionated choice to exclude internal/global macros from the dbt-docs site; they're included in `manifest.json`, just like every other macro. I'd like to document global macros (perhaps a community-sourced effort??), which we can use to power other cool metadata-y things down the line